CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changelog
```

Comment:
Is this files supposed to be added here? It's already in master.

Reply:
I did a rebase with master at some point and now I have in develop as well 😅  Guess it's not causing any harm, is it? 🥺

Reply:
I did a rebase with master at some point and now I have in develop as well 😅  Guess it's not causing any harm, is it? 🥺